utility/src/main/java/org/oppia/android/util/parser/html/HtmlParser.kt

```diff
@@ -163,7 +189,30 @@ class HtmlParser private constructor(
         entityId,
         imageCenterAlign,
         consoleLogger,
-        customOppiaTagActionListener
+        customOppiaTagActionListener,
+        policyOppiaTagActionListener
+      )
+    }
+
+    /**
+     * Returns a new [HtmlParser] with an optionally specified [CustomOppiaTagActionListener] and
+     * [PolicyOppiaTagActionListener] for handling custom Oppia tag events. Note that Oppia image
+     * loading is specifically not supported (see the other [create] method if image support is
+     * needed).
+     */
+    fun create(
```

Comment:
Are there tests for this factory method to specifically verify that image loading isn't supported?

It also seems like it just passes empty GCS and entity ids rather than disabling images. I'd expect the current code to still attempt to create drawable spans for images even though it shouldn't since they should be disabled.

Reply:
How can I disable it? I am not clear here.

Reply:
Done. Added secondary constructor for html parser  one with image loading and another without image support.

Reply:
Initially, I did it by passing empty strings and null for GCS and entity ids but you said it doesn't seem to be disable the image loading. Hence, I created 2 methods and constructors to completely remove image loading.

No specific tests are added to check this ideally. But there are tests that does not take image parameters.

Reply:
I made UrlImageParser nullable and passing the other image loading entities to pass empty strings.

Reply:
Sorry to be really clear: I didn't mean to remove the create (having 2 makes sense). I meant that we should have tests for each create to make sure it creates the correct kind of parser, and have only 1 constructor (rather than 2) that relies on a null ``UrlImageParserFactory`` being passed in.

Reply:
There are testcases with 2 create methods.It is tested in OnboardingFragmentTest and HtmlParserTest.
- OnboardingFragmentTest tests this create method without loading image. 
- HtmlParserTest tests this create method with image loading.

This would be enough to test these methods?

Reply:
We should have tests in HtmlParserTest to explicitly verify that each create method creates the parser with correct support (i.e. the one that creates an HtmlParser to not support images should explicitly verify that). I wouldn't expect OnboardingFragmentTest to be responsible for this since it's meant to verify the UI and behaviors of OnboardingFragment.